liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/utils/MavenProjectUtil.java

```diff
@@ -61,41 +62,30 @@ public static String getPluginExecutionConfiguration(MavenProject proj, String p
     }
 
     /**
-     * Get a nested configuration value from a plugin
+     * Get directory and targetPath configuration values from the Maven WAR plugin
      * @param proj the Maven project
-     * @param pluginGroupId the plugin group id
-     * @param pluginArtifactId the plugin artifact id
-     * @param key1 the configuration key to get from
-     * @param key2 the configuration key nested in key1 to get from or null
-     * @param childName the configuration key nested in key2 to get from or null
-     * @return the array of values corresponding to the configuration keys
+     * @return a Map of source and target directories corresponding to the configuration keys
```

Comment:
In the Javadoc comments, describe the conditions where this would return null.

Reply:
Agreed